README.rst

```diff
@@ -194,7 +200,9 @@ This backend can be run in serial or blocked mode and defaults to running in the
 if ``/cpu/self/memcheck`` is selected at runtime.
 
 The ``/*/occa`` backends rely upon the `OCCA <http://github.com/libocca/occa>`_ package to provide
-cross platform performance.
+cross platform performance. Additionally, users can pass specific device flags such as:
```

Comment:
Thanks @dmed256 do you think you could add a brief sentence to the README for documentation, like it was done recently for the [MAGMA backend](https://github.com/CEED/libCEED/commit/2d4794f3e2a4414a4a5fb41264405bb4c5517adf)?